models/sources/src_history_transactions.yml

```diff
@@ -22,25 +22,25 @@ sources:
             tests:
               - unique:
                   config:
-                    where: closed_at >= datetime_trunc(datetime_sub(current_datetime(), interval 2 day), day)
-                            and closed_at < datetime_trunc(current_datetime(), day)
+                    where: cast(closed_at as datetime) >= datetime_trunc(datetime_sub(current_datetime(), interval 2 day), day)
+                            and cast(closed_at as datetime) < datetime_trunc(current_datetime(), day)
```

Comment:
might need to add the partition fields to this where clause otherwise it's gonna be expensive

Reply:
Yeah I'll look at these tables again. It should be based on the partitioned field